e2e-playwright/actions/homePage.ts

```diff
@@ -0,0 +1,12 @@
+import { Page, expect } from '@playwright/test';
+
+const openHomePage = async (page: Page) => {
+  await page.goto('/');
```

Comment:
It'd be good to have initialized enum with routes like `AppRoute` or so, including `AppRoute.Home` of course 😀

Reply:
Yeah definitely good idea! I switched string for `AppRoute` enum.